### PR TITLE
RES-005-landingPage-client

### PR DESCRIPTION
**Pull request by DonataB**

### CODE REVIEW

**Review by MateuszWyborski — DISMISSED** [review]
Wydaje się wszystko w porządku

File: src/components/LandingPage/LandingPageImg/LandingPageImg.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import image from "./bg-landingPage.png";
```
Comment (lagdush) [inline src/components/LandingPage/LandingPageImg/LandingPageImg.js]: First, the file size is huge. Secondly, the image has low resolution. It needs to be corrected.

File: src/components/LandingPage/LandingPageBody/LandingPageBody.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import { Card } from "react-bootstrap";
+
+const LandingPageBody = () => {
+    return (
+        <>
+
+        <Card
+        className='bg-transparent text-white text-center'
+        style={{ position: "absolute", top: "260px" }}>
```
Comment (lagdush) [inline src/components/LandingPage/LandingPageBody/LandingPageBody.js]: Try to create a responsive view. Use flex and grid to create layouts.

**Review by lagdush — CHANGES_REQUESTED** [review]

File: src/components/LandingPage/LandingPageImg/LandingPageImg.js (hunk 1 above)
Comment (DonataB) [reply src/components/LandingPage/LandingPageImg/LandingPageImg.js]: Problem with the size and resolution is solved.

Comment (lagdush) [reply src/components/LandingPage/LandingPageImg/LandingPageImg.js]: Unfortunately it is not. And still no responsiveness in this view.
Zrzut ekranu 2022-01-22 o 00 07 36

**Review by lagdush — CHANGES_REQUESTED** [review]
We have such a problem...

https://user-images.githubusercontent.com/38866668/150833276-843a1bc0-58ed-4044-a81c-36241050143f.mov

File: src/components/LandingPage/LandingPage.js [hunk 3]
```diff
@@ -1,11 +1,51 @@
...
+import "bootstrap/dist/css/bootstrap.min.css";
 
 const LandingPage = () => {
-  return (
-    <div style={{ height: "100vh" }}>
-      <div>LandingPage</div>
-    </div>
-  );
+	return (
+		<>
```
Comment (lagdush) [inline src/components/LandingPage/LandingPage.js]: You don't need to use a fragment when you have a parent element.

File: src/components/LandingPage/LandingPage.js [hunk 4]
```diff
@@ -1,11 +1,51 @@
...
+			<div
+				style={{
+					backgroundImage: `url(${background})`,
+					backgroundRepeat: "no-repeat",
+					backgroundPosition: "center",
+					backgroundSize: "cover",
+					height: "100vh",
+					overflowY: "auto",
+				}}>
```
Comment (lagdush) [inline src/components/LandingPage/LandingPage.js]: I'm sorry but you haven't solved the problem. What is the purpose of setting overflowY to auto? Maybe you need to use a different value here?

**Review by lagdush — CHANGES_REQUESTED** [review]

File: src/components/LandingPage/LandingPage.js (hunk 4 above)
Comment (DonataB) [reply src/components/LandingPage/LandingPage.js]: It was by mistake when I tried to see a double scroll

File: src/components/LandingPage/LandingPage.js (hunk 3 above)
Comment (DonataB) [reply src/components/LandingPage/LandingPage.js]: Ok, I'm already fixing it

**Review by lagdush — APPROVED** [review]

**Review by EMEZYK — APPROVED** [review]

**Pull request merged**